.github/workflows/dev-test.yml

```diff
@@ -60,6 +60,9 @@ jobs:
       - name: Install Dependencies
         run: yarn install --immutable
 
+      - name: Build Package
+        run: yarn build --clean
```

Comment:
If the tests need build, it should be in prod test.

Reply:
https://github.com/prettier/prettier/blob/5f7aedc1cf0a1b3b4ec2c5a5ca0c09d1e0d00660/jest.config.js#L34

Reply:
Aha! So _that's_ why there was some commented out stuff with an `isProduction` variable in the test before! I had not made the connection that "production" tests were ones where `yarn build` had been run.

Change made; should be ready for re-review.